apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Snippets/DataModelSnippets.cs

```diff
@@ -272,5 +272,39 @@ public CustomCity FromFirestore(object value)
             }
         }
         // End sample
+
+        // Sample: DocumentId
+        [FirestoreData]
+        public class ChatRoom
+        {
+            //[FirestoreDocumentId]
```

Comment:
I would guess this isn't meant to be commented?

Reply:
Gah! That was to prove the test was effective. Thank you!